openpype/modules/webserver/webserver_cli.py

```diff
@@ -0,0 +1,151 @@
+import time
+import os
+from datetime import datetime
+import requests
+import json
+
+from openpype.lib import PypeLogger
+
+from .webpublish_routes import (
+    RestApiResource,
+    OpenPypeRestApiResource,
+    WebpublisherBatchPublishEndpoint,
+    WebpublisherTaskPublishEndpoint,
+    WebpublisherHiearchyEndpoint,
+    WebpublisherProjectsEndpoint,
+    BatchStatusEndpoint,
+    PublishesStatusEndpoint
+)
+
+from openpype.api import get_system_settings
+
+SERVER_URL = "http://172.17.0.1:8079"  # machine is not listening on localhost
```

Comment:
If it's `cli` why not add option to change port?

Reply:
The same for host and maybe for schema but I would say that port is important as it's easy to have `8079` occupied.